x-pack/plugins/case/server/connectors/case/index.ts

```diff
@@ -69,18 +73,17 @@ async function executor(
   const { subAction, subActionParams } = params;
   let data: CaseExecutorResponse | null = null;
 
-  const { savedObjectsClient } = services;
-  const caseClient = createCaseClient({
+  const { savedObjectsClient, scopedClusterClient } = services;
+  const caseClient = createExternalCaseClient({
     savedObjectsClient,
-    request: {} as KibanaRequest,
-    response: kibanaResponseFactory,
+    scopedClusterClient,
+    // we might want the user information to be passed as part of the action request
+    user: nullUser,
```

Comment:
I found out that you can `scopedClusterClient.security.getUser()`. Just for future reference in case we need it.

Reply:
oh interesting! I'll add it to the list of stuff to refactor 👍